src/editor/wfstransaction.js

```diff
@@ -0,0 +1,103 @@
+/* ========================================================================
+ * Copyright 2016 Origo
+ * Licensed under BSD 2-Clause (https://github.com/origo-map/origo/blob/master/LICENSE.txt)
+ * ======================================================================== */
+"use strict";
+
+var ol = require('openlayers');
+var $ = require('jquery');
+var viewer = require('../viewer');
+var editsStore = require('./editsstore')();
+var dispatcher = require('./editDispatcher');
+
+var format = new ol.format.WFS();
+var serializer = new XMLSerializer();
+
+module.exports = function(transObj, layerName) {
+  wfsTransaction(transObj, layerName);
+}
+
+function writeWfsTransaction(transObj, options) {
+  var node = format.writeTransaction(transObj.insert, transObj.update, transObj.delete, options);
+  return node;
+}
+
+function wfsTransaction(transObj, layerName) {
+  var srsName = viewer.getProjectionCode();
+  var layer = viewer.getLayer(layerName);
+  var featureType = layer.get('featureType');
+  var source = viewer.getMapSource()[layer.get('sourceName')];
+  var options = {
+    gmlOptions: {
+      srsName: srsName
+    },
+    featureNS: source.workspace,
+    featurePrefix: source.prefix || source.workspace,
+    featureType: featureType
+  };
+  var node = writeWfsTransaction(transObj, options);
+  $.ajax({
+    type: "POST",
+    url: source.url,
+    data: serializer.serializeToString(node),
+    contentType: 'text/xml',
+    success: success,
+    error: error,
+    context: this
+  });
+
+  function success(data) {
+    var result = readResponse(data);
+    if (result) {
+      if (result.transactionSummary.totalUpdated > 0) {
+        dispatcher.emitChangeFeature({
+          feature: transObj.update,
+          layerName: layerName,
+          status: 'finished',
+          action: 'update'
+        });
+      }
+      if (result.transactionSummary.totalDeleted > 0) {
+        dispatcher.emitChangeFeature({
+          feature: transObj.delete,
+          layerName: layerName,
+          status: 'finished',
+          action: 'delete'
+        });
+      }
+      if (result.transactionSummary.totalInserted > 0) {
+        var feature = transObj.insert;
+        dispatcher.emitChangeFeature({
+          feature: transObj.insert,
+          layerName: layerName,
+          status: 'finished',
+          action: 'insert'
+        });
+        var insertIds = result.insertIds;
+        insertIds.forEach(function(insertId, index) {
+          feature[index].setId(insertId);
+        });
+      }
+    } else {
+      error();
+    }
+  }
+
+  function error(e) {
+    var errorMsg = e ? (e.status + ' ' + e.statusText) : "";
+    alert('Det inträffade ett fel när ändringarna skulle sparas till servern...<br><br>' +
```

Comment:
<br><br> does display in full text when you get the error message. I think we should replaced it with newline (\n): 
'Det inträffade ett fel när ändringarna skulle sparas till servern...\n'